README.md

```diff
@@ -83,3 +89,13 @@ fail with: ```docker manifest is only supported when experimental cli features
 are enabled```. Checkout
 [this article](https://medium.com/@mauridb/docker-multi-architecture-images-365a44c26be6)
 to understand why and how to fix it.
+
+The above steps is a little bit different for `riscv64` platform:  
```

Comment:
We should explain _why_

Reply:
Resolved by adding more detailed explanation in README.md